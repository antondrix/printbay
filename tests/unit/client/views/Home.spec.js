import createWrapper from "../wrapper";
import Home from "@views/Home";

jest.mock("@/store/models/Item", () => require("../mocks/Item"));

describe("Home.vue", () => {
  it("should render correctly", () => {
    const wrapper = createWrapper(Home);
    expect(wrapper).toMatchSnapshot();
  });
});
