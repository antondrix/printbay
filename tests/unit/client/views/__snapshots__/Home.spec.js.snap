// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home.vue should render correctly 1`] = `
<v-layout-stub tag="div" column="" class="[object Object]">
  <v-layout-stub tag="div">
    <h1>Featured Prints</h1>
  </v-layout-stub>
  <v-divider-stub class="mt-2 mb-4"></v-divider-stub>
  <v-layout-stub tag="div">
    <v-container-stub tag="div" fluid="" grid-list-lg="" class="px-0">
      <v-layout-stub tag="div" row="" wrap="" class="item-summaries">
        <itemsummary-stub item="[object Object]"></itemsummary-stub>
        <itemsummary-stub item="[object Object]"></itemsummary-stub>
      </v-layout-stub>
    </v-container-stub>
  </v-layout-stub>
</v-layout-stub>
`;
